### PR TITLE
fix: height on body

### PR DESCRIPTION
**Pull request by afonsopimenta**

### CODE REVIEW

**Comment by ruilopesm** [comment]
Explain the change 👀

**Review by diogogmatos — DISMISSED** [review]

**Comment by afonsopimenta** [comment]
@diogogmatos told me there was a problem on mobile where the page would have scroll due to the height of the search bar on certain browsers.
This fixes that unintended scroll.

**Review by ruilopesm — APPROVED** [review]

File: pre
Comment (ruilopesm) [inline pre:1]: What is this file intended for?

Comment (afonsopimenta) [reply pre:1]: Not supposed to be there, i deleted it but git must not have. Deletting it now.
